packages/web-worker/tsconfig.build.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../../tsconfig.json",
+  "compileOnSave": false,
+  "compilerOptions": {
+    "outDir": "./dist",
+    "baseUrl": "./src",
+    "rootDir": "./src"
+  },
+  "include": [
+    "../../config/typescript/**/*",
```

Comment:
Are we pulling in anything from here? If not we can 🔥

Reply:
It's part of the template, I feel like it would be weird to remove